### DISCUSSION TITLE
Pembentukan regression model

### DISCUSSION THREAD

**Discussion by AGelfP** [teamalgoritma/community#524] [Q&A] [answered]
Saya berencana untuk membentuk model regresi untuk melihat hubungan pengaruh Market Return, Size Factor, Firm Value Factor, dan Trading Volume terhadap Excess Return saham dengan menggunakan data historis dari 113 saham sebagai sample menggunakan R. Apakah pembentukan regresi harus dibentuk satu per satu untuk setiap saham atau apakah bisa dibentuk 1 model regresi yang menjelaskan pergerakan semua 113 saham tersebut?

**Comment by muhrisman** [comment, marked as answer]
Izin bantu jawab, modelnya cukup satu saja pak untuk mewakili semuanya kecuali bapak ingin membuat model regresi per nama saham itu juga bisa.

Namun perlu diingat untuk membuat model regresi tidak bisa menggunakan satu baris observasi saja, bapak bisa perbanyak sampel untuk masing-masing nama saham jika ingin menggunakan teknik tersebut.

Best Regards,
Muhamad Risman
